README.md

```diff
@@ -1,20 +1,41 @@
 # pipelines-api-examples
 
-This repository contains examples for the Pipelines API.
+This repository contains examples for the [Google Genomics Pipelines API]
+(https://cloud.google.com/genomics/reference/rest/v1alpha2/pipelines).
 
-The pipelines API is an easy way to create, run, and monitor command-line tools on Google Compute Engine in a Docker container. You can use it like you would a job scheduler. Input files are copied from a Cloud Storage bucket to a local disk, and output files are copied back to Cloud Storage. You can submit batch operations from your laptop, and have them run in the cloud, using any tools that you can package up with Docker, or that someone has already packaged up and shared.
+The API provides an easy way to create, run, and monitor command-line tools
+on Google Compute Engine running in a Docker container. You can use it like
+you would a job scheduler.
+
+The most common use case is to run an off-the-shelf tool or custom script
+that reads and writes files. You may want to run such a tool over files in
+Google Cloud Storage. You may want to run this independently over hundreds
+or thousands of files.
+
+The typical flow for a pipeline is:
+
+  1. Create a Compute Engine virtual machine
+  1. Copy one or more files from Cloud Storage to a disk
+  1. Run the tool on the file(s)
+  1. Copy the output to Cloud Storage
+  1. Destroy the Compute Engine virtual machine
```

Comment:
Hi Matt,

Learned something new today about Github rendering :)  Didn't know that a list of the same number will actually render as a sequence of numbers.

Thanks,
~p